src/transaction/log_applier.c

```diff
@@ -7043,141 +7043,177 @@ la_print_log_arv_header (const char *database_name, LOG_ARV_HEADER * hdr, bool v
-	  printf ("%-30s : %s\n", "Log committed time", timebuf);
+      db_datetime_to_string ((char *) timebuf, 1024, &ha_apply_info.last_access_time);
+      printf ("%-30s : %s\n", "Last access time", timebuf);
+    }
 
-	  db_datetime_to_string ((char *) timebuf, 1024, &ha_apply_info.last_access_time);
-	  printf ("%-30s : %s\n", "Last access time", timebuf);
-	}
+  printf ("%-30s : %ld\n", "Insert count", ha_apply_info.insert_counter);
+  printf ("%-30s : %ld\n", "Update count", ha_apply_info.update_counter);
+  printf ("%-30s : %ld\n", "Delete count", ha_apply_info.delete_counter);
+  printf ("%-30s : %ld\n", "Schema count", ha_apply_info.schema_counter);
+  printf ("%-30s : %ld\n", "Commit count", ha_apply_info.commit_counter);
+  printf ("%-30s : %ld\n", "Fail count", ha_apply_info.fail_counter);
 
-      printf ("%-30s : %ld\n", "Insert count", ha_apply_info.insert_counter);
-      printf ("%-30s : %ld\n", "Update count", ha_apply_info.update_counter);
-      printf ("%-30s : %ld\n", "Delete count", ha_apply_info.delete_counter);
-      printf ("%-30s : %ld\n", "Schema count", ha_apply_info.schema_counter);
-      printf ("%-30s : %ld\n", "Commit count", ha_apply_info.commit_counter);
-      printf ("%-30s : %ld\n", "Fail count", ha_apply_info.fail_counter);
+  if (verbose)
+    {
+      db_datetime_to_string ((char *) timebuf, 1024, &ha_apply_info.start_time);
+      printf ("%-30s : %s\n", "Start time", timebuf);
+    }
 
-      if (verbose)
+  if (check_replica_info)
+    {
+      replica_time_bound_str = prm_get_string_value (PRM_ID_HA_REPLICA_TIME_BOUND);
+      db_datetime_to_string2 ((char *) timebuf, 1024, &ha_apply_info.log_record_time);
+
+      printf ("\n *** Replica-specific Info. *** \n");
+      if (replica_time_bound_str == NULL)
 	{
-	  db_datetime_to_string ((char *) timebuf, 1024, &ha_apply_info.start_time);
-	  printf ("%-30s : %s\n", "Start time", timebuf);
+	  printf ("%-30s : %d second(s)\n", "Deliberate lag", prm_get_integer_value (PRM_ID_HA_REPLICA_DELAY_IN_SECS));
 	}
 
-      if (check_replica_info)
+      printf ("%-30s : %s\n", "Last applied log record time", timebuf);
+      if (replica_time_bound_str != NULL)
 	{
-	  replica_time_bound_str = prm_get_string_value (PRM_ID_HA_REPLICA_TIME_BOUND);
-	  db_datetime_to_string2 ((char *) timebuf, 1024, &ha_apply_info.log_record_time);
-
-	  printf ("\n *** Replica-specific Info. *** \n");
-	  if (replica_time_bound_str == NULL)
-	    {
-	      printf ("%-30s : %d second(s)\n", "Deliberate lag",
-		      prm_get_integer_value (PRM_ID_HA_REPLICA_DELAY_IN_SECS));
-	    }
-
-	  printf ("%-30s : %s\n", "Last applied log record time", timebuf);
-	  if (replica_time_bound_str != NULL)
-	    {
-	      printf ("%-30s : %s\n", "Will apply log records up to", replica_time_bound_str);
-	    }
+	  printf ("%-30s : %s\n", "Will apply log records up to", replica_time_bound_str);
 	}
     }
 check_applied_info_end:
-  if (error != NO_ERROR)
+  if (error != NO_ERROR && res != 0)
```

Comment:
`&& res != 0` is added. Because when res == 0, `er_set()` does not work and error message is printed already.

Reply:
Why doesn't it work? `ER_GENERIC_ERROR` is set when res is 0.

Reply:
It would be easier to understand to move the part that outputs this error to the place where the error occurs, 
and simply return an error from this place.

Reply:
@wrlawodms Because error message is printed when the error is set to `ER_GENERIC_ERROR`. So this is not necessary to work res = 0. Is this right you to ask me?

Reply:
@joohok 
```suggestion
  if (!res)
    return error;
  if (error != NO_ERROR )
```

I think you suggest like this way?

Reply:
How about just returning ER_FAILED at 7090L?

Reply:
@wrlawodms Original flow was that position. But I think there should be consistent flow.
(flow is goto check_applied_info_end;) 

@joohok what do you think in this flow?

Reply:
```suggestion
  return error;
```

just set error where it occurred, and return error code at this label.